src/components/SearchCard/SearchCardMeta.js

```diff
@@ -0,0 +1,66 @@
+import { formatString } from 'helpers/formatString'
+import { formatPlatformName } from 'helpers/formatPlatformName'
+import { Box, Text } from 'grommet'
+import { IconBadge } from 'components/shared/IconBadge'
+
+export const SearchCardMeta = ({
+  metaData: { downloadableSamples, organismNames, platformNames, technology }
```

Comment:
```suggestion
  metadata: { downloadableSamples, organismNames, platformNames, technology }
```

I'd like to use words like `metadata` and `dataset` as a single word to prevent multiple uses across the site

Reply:
Done, thank you!